api/users/serializers.py

```diff
@@ -117,6 +122,21 @@ def get_absolute_url(self, obj):
             'version': self.context['request'].parser_context['kwargs']['version']
         })
 
+    def get_node_count(self, obj):
+        return obj.nodes.count()
```

Comment:
We need to make sure these counts match what you would see if you fetched the relationship directly.  user.nodes.count(), for example, will include all nodes, registrations, quickfilenodes, deleted nodes, etc.  Look at how the query is being built on `get_default_queryset` on UserNodes.